src/plotter/collection.py

```diff
@@ -140,68 +153,154 @@ def norm_ds(self, th: TH1, ds: dataset, norm: normalizationHelper):
         if norm.byLumi:
             th.Scale(ds.lumi)
 
+class SuperCollection:
+    """ Holds set of collections or SuperCollections, necessary for scaling collections"""
+        
+    def __init__(self, title: str, scale_factor : Optional[float] = 1):
+        self.container: Dict[str, Union[collection, 'SuperCollection']] = []
```

Comment:
```suggestion
        self.container: List[Union[collection, 'SuperCollection']] = []
```
Actually surprised this did not trigger error in mypy...